cvmfs/server/cvmfs_server_sys.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# This file is part of the CernVM File System
+# This script takes care of creating, removing, and maintaining repositories
+# on a Stratum 0/1 server
+
+# This file should implement all platform specific functions. The other components of the
+# cvmfs_server script should interact with the underlying system only through the functions
+# defined here. This allows simple(r) unit testing of the "cvmfs_server" script
+
+# Returns the major.minor.patch-build kernel version string
+cvmfs_sys_uname() {
+    uname -r | grep -oe '^[0-9]\+\.[0-9]\+.[0-9]\+-*[0-9]*'
+}
+
+
+cvmfs_sys_is_regular_file() {
```

Comment:
I went with your suggestion, which maps to a namespacing:: `cvmfs::sys::file::is_regular`. We should keep this convention, when possible.